src/node_sqlite.cc

```diff
@@ -585,27 +589,77 @@ bool DatabaseSync::ShouldIgnoreSQLiteError() {
   return ignore_next_sqlite_error_;
 }
 
+bool ValidateDatabasePath(Environment* env,
```

Comment:
This field_name was added because in DatabaseSync the name is `location` and in backup, it is `destination`.

Hopefully, we can simplify this and call both path. WDYT?